frontend_tests/casper_tests/13-user-deactivation.js

```diff
@@ -85,21 +85,21 @@ casper.then(function () {
 
 casper.then(function () {
     casper.waitUntilVisible(user_row('default-bot'), function () {
-        casper.test.assertSelectorHasText(user_row('default-bot'), 'Deactivate');
+        casper.test.assertSelectorHasText(user_row('default-bot'), '');
         casper.click(user_row('default-bot') + ' .deactivate');
     });
 });
 
 casper.then(function () {
     casper.waitUntilVisible(user_row('default-bot') + '.deactivated_user', function () {
-        casper.test.assertSelectorHasText(user_row('default-bot'), 'Reactivate');
+        casper.test.assertSelectorHasText(user_row('default-bot'), '');
         casper.click(user_row('default-bot') + ' .reactivate');
     });
 });
 
 casper.then(function () {
     casper.waitUntilVisible(user_row('default-bot') + ':not(.deactivated_user)', function () {
-        casper.test.assertSelectorHasText(user_row('default-bot'), 'Deactivate');
+        casper.test.assertSelectorHasText(user_row('default-bot'), '');
```

Comment:
It feels like these tests aren't really testing anything now; can we easily check the title attribute?  Or barring that, just check the CSS classes for `fa-user-plus` vs. `fa-user-times`?

Reply:
ok sure